backend/src/ipfs.ts

```diff
@@ -0,0 +1,59 @@
+import { Request, Response } from 'express';
+import { createHeliaHTTP } from '@helia/http';
+import { getStarknetId } from './starknet';
+import axios from 'axios';
+
+const PINATA_API_KEY = 'your_api_key';
+const PINATA_SECRET_API_KEY = 'your_secret_api_key';
+
+const pinToPinata = async (ipfsHash: string) => {
+  const url = "https://api.pinata.cloud/pinning/pinByHash";
+  const headers = {
+    "pinata_api_key": PINATA_API_KEY,
+    "pinata_secret_api_key": PINATA_SECRET_API_KEY
+  };
+  const payload = {
+    "hashToPin": ipfsHash
+  };
+  return axios.post(url, payload, { headers });
+};
+
+let helia: any;
+
+export const initializeHelia = async () => {
+  helia = await createHeliaHTTP();
```

Comment:
Are you sure that Helia does not initialize a full node (one that connects to DHT)? We don't want that